mission/functions/systems/supplies/fn_create_supply_officer.sqf

```diff
@@ -40,6 +40,7 @@ if (isNull _officer) then {
 
 _officer setVariable ["vn_mf_supply_drop_marker", _dropMarker, true];
 
+// @dijksterhuis remoteExec needing fiing
```

Comment:
the remote exec below is done by the server, so not something we need to worry about with `CfgRemoteExec` (server is always allowed to do whatever it wants with remote execs)